src/features/debugger/debugMessage.ts

```diff
@@ -172,12 +174,20 @@ async function debugMessageDelivery(
     const errorReason = extractReasonString(err);
     logger.debug(errorReason);
 
-    const bytecodeHasHandle = await tryCheckBytecodeHandle(destProvider, recipient);
-    if (!bytecodeHasHandle) {
-      logger.info('Bytecode does not have function matching handle sig');
+    // const bytecodeHasHandle = await tryCheckBytecodeHandle(destProvider, recipient);
+    // if (!bytecodeHasHandle) {
+    //   logger.info('Bytecode does not have function matching handle sig');
+    //   return {
+    //     status: MessageDebugStatus.RecipientNotHandler,
+    //     description: `Recipient contract should have handle function of signature: ${HANDLE_FUNCTION_SIG}. Check that recipient is not a proxy. Error: ${errorReason}`,
+    //     calldataDetails,
+    //   };
+    // }
+
+    if (debugIgnoredChains.includes(destName)) {
       return {
-        status: MessageDebugStatus.RecipientNotHandler,
-        description: `Recipient contract should have handle function of signature: ${HANDLE_FUNCTION_SIG}. Check that recipient is not a proxy. Error: ${errorReason}`,
+        status: MessageDebugStatus.MessageNotDelivered,
+        description: 'Message not delivered, there may be an error',
```

Comment:
Rather than create this new case that doesn't really communicate anything, I'd prefer if you just never called  `debugMessage` for the blacklisted chains please